widgets/InboxMessagePreview.php

```diff
@@ -71,6 +71,10 @@ private function lastParticipant(): ?User
     private function getUsername(): string
     {
         $user = $this->lastParticipant();
+        if (!$user) {
+            return Yii::t('MailModule.base', 'Deleted user');
```

Comment:
@yurabakhtin Hmm, the mail messages of "Hard Deleted" users should also be deleted.

For SoftDeleted users, "getDisplayName()" should always return something.

Reply:
@luke- For me the error looks strange, because I too cannot understand when the last participant may be null.
I think we should apply this fix in order to see when users will see the text "Deleted user" in order to fix this later.

Reply:
@luke- Yes, the integrity check should delete a message without user - https://github.com/humhub/mail/blob/master/Events.php#L69-L76